bin/function_override_wrapper.sh.erb

```diff
@@ -1,6 +1,9 @@
 /usr/bin/env bash -c '
 # load in command and function overrides
-for override_file in <%=function_override_path_binding_for_template%>; do
+system_ls=$(which ls)
+system_grep=$(which grep)
```

Comment:
If we use `whereis` instead of `which`, will that protect us against the case where absolute paths stub out ls/grep (since `which` searches PATH)?